zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -117,10 +115,10 @@ fn generate_accounts_with_assets<T: Config>(
     for i in 0..acc_total {
         let acc = account("AssetHolder", i, 0);
         if mut_acc_asset > 0 {
-            T::AssetManager::deposit(asset, &acc, min_liquidity)?;
+            T::AssetManager::deposit(asset, &acc, min_liquidity).unwrap();
```

Comment:
In general should not we prefer to avoid unwarp() when function returns Result<> ?